src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -128,7 +129,7 @@ static void LogRequest(object sender, EventArgs e)
         {
             if (!_isEnabled) return;
 
-            var request = HttpContext.Current.Request;
+            var request = HttpContextCurrent.Request;
             Logger.Write(_requestLoggingLevel, "HTTP {Method} for {RawUrl}", request.HttpMethod, request.RawUrl);
```

Comment:
This is beyond the scope of the original problem and original fix, but I don't see a problem of fixing this on the way.